library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayRecorder.kt

```diff
@@ -60,6 +61,15 @@ internal class SessionReplayRecorder : OnWindowRefreshedCallback, Recorder {
         TimeUnit.MILLISECONDS,
         LinkedBlockingDeque()
     )
+
+    private val blockingQueueExecutorService = ThreadPoolExecutor(
```

Comment:
I would rather use `LoggingThreadPoolExecutor`. The thing is that if task submitted to `ThreadPoolExecutor` throws and we don't unwrap the resulting `Future`, exception will be lost. While with `LoggingThreadPoolExecutor` it will be logged.

Reply:
I've added a todo to change to `LoggingThreadPoolExecutor` once V2 is merged